src/main/java/com/bullhornsdk/data/api/BullhornRestCredentials.java

```diff
@@ -82,10 +103,16 @@ public void setRestAuthorizeUrl(String restAuthorizeUrl) {
 		this.restAuthorizeUrl = restAuthorizeUrl;
 	}
 
+	public boolean credentialCheck() {
```

Comment:
Check() is too ambiguous of a term - use something that is more descriptive, like `hasValidCredentials`

Reply:
Fair point. I wonder if `hasValidCredentials` is still a little too ambiguous.

I think the naming convention here should specify what the intentions of the method actually are.

`hasSessionProvided` maybe?

Reply:
True, still ambiguous. How about just removing it entirely, and having the RestApiSession be explicit:
```
if (bullhornRestCredentials.restUrl != null && bullhornRestCredentials.bhRestToken != null) {
```

Reply:
We'd be repeating code in a few places if we did that. It can be done, but I was going for readability.

Reply:
Replaced this with `hasNoSessionProvided`. I think that's a bit more descriptive.